pkg/controller.v1beta1/tensorflow/controller.go

```diff
@@ -368,11 +368,6 @@ func (tc *TFController) reconcileTFJobs(tfjob *tfv1beta1.TFJob) error {
 			}
 		}
 
-		// Initialize the status.
-		initializeTFReplicaStatuses(tfjob, tfv1beta1.TFReplicaTypeWorker)
-		initializeTFReplicaStatuses(tfjob, tfv1beta1.TFReplicaTypePS)
-		initializeTFReplicaStatuses(tfjob, tfv1beta1.TFReplicaTypeChief)
-		initializeTFReplicaStatuses(tfjob, tfv1beta1.TFReplicaTypeMaster)
 		return tc.updateStatusHandler(tfjob)
```

Comment:
If we remove initialize, do we still need to update here?

Reply:
Removed